_sass/_the-plan.scss

```diff
@@ -1,4 +1,30 @@
 /** 
+ * Mayday Page Styles
+ */
+#top-space-h1 {
+  margin-bottom: .415em;
+}
+
+#plan-byline{//the plan-byline and progress-clearance are just to make sure the typography remains correct
```

Comment:
One key thing though, Generally, lets try to avoid styling based on IDs.  If you have some down time, let's try to address it.